### ISSUE TITLE
Ignore functions regex

### ISSUE THREAD

**Issue by silverwind** [AndyOGo/stylelint-declaration-strict-value#324]
I would like to allow certain functions like [`color-mix`](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color-mix), but it seems the current `ignoreFunction` option is not flexible enough.

This should be flagged:
```css
color: rgb(255 0 0);
```

This should be allowed:
```css
color: color-mix(in srgb, var(--color-red), var(--color-blue));
```

Maybe have an `ignoreFunctionRegex: /^color-mix$/` option?

**Comment by Zeeex** [comment]
I would also like this option, and you could probably re-use the same keyword, like for "ignoreValues"

![image](https://github.com/AndyOGo/stylelint-declaration-strict-value/assets/16362320/bcb32cf1-14a9-470c-bf8c-7028112120f2)
Something like this:
```
"ignoreFunctions": {
	"": false,
	"/RegExp/[darken]": true,
	"/RegExp/[lighten]": true
},
```
